data/campaigns/Heir_To_The_Throne/achievements.cfg

```diff
@@ -77,6 +77,100 @@
         hidden_hint= _ "Embrace your inner Paladin yet again"
     [/achievement]
 
+    [achievement]
+        id="HttT_S10_Egg_Snatcher"
+        name= _ "Egg Snatcher"
+        name_completed= _ "Egg Snatcher"
+        description= _ "Steal the Mother Gryphon's Eggs"
+        description_completed= _ "Shameless Stole the Mother Gryphon's Egg"
+        icon="scenery/nest-full.png~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="scenery/nest-empty.png~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S10_Master_Pacifist"
+        name= _ "Master Pacifist"
+        name_completed= _ "Master Pacifist"
+        description= _ "Do not antagonize the Gryphons"
+        description_completed= _ "Did not antagonize the Gryphons"
+        icon="units/monsters/gryphon.png~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="units/monsters/gryphon.png~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        hidden=yes
+        hidden_name= _ "???"
+        hidden_hint= _ "Violence is not the answer"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S11_Strike_First_Strike_Hard"
+        name= _ "Strike First Strike Hard"
+        name_completed= _ "Struck First Struck Hard"
+        description= _ "Defeat the orcish commander at the northern banks of Abez"
+        description_completed= _ "Defeated the orcish commander at the northern banks of Abez"
```

Comment:
I believe "bank" should be plural here, and "on" the bank.

Reply:
I still think "at the bank" sounds strange, shouldn't it be "on the northern banks"?

Reply:
Oops..need to still patch the preposition thing